PyOptionalOutputToolExample/PyOptionalOutputToolExampleEngine.py

```diff
@@ -0,0 +1,170 @@
+import AlteryxPythonSDK as Sdk
+import xml.etree.ElementTree as Et
+
+
+class AyxPlugin:
+    def __init__(self, n_tool_id, alteryx_engine, generic_engine, output_anchor_mgr):
+        """Initializing members that will be used."""
+
+        # Miscellaneous properties
+        self.n_tool_id = n_tool_id
+        self.name = str('PyOptionalOutputToolExample_') + str(self.n_tool_id)
+        self.initialized = False
+
+        # Engine handles
+        self.alteryx_engine = alteryx_engine
+        self.generic_engine = generic_engine
+
+        # Output anchor management
+        self.output_anchor_mgr = output_anchor_mgr
+        self.output_anchor = None
+
+        # Default configuration setting
+        self.message = Sdk.EngineMessageType
```

Comment:
I'm not sure this assignment makes much sense... `None` or a default member of the enum would make more sense.